generic_tracers/generic_COBALT.F90

```diff
@@ -536,129 +536,252 @@ subroutine user_add_params(param_file)
+    call get_param(param_file, "generic_COBALT", "k_sio4_Lg",phyto(LARGE)%k_sio4, &
+                   "half-saturation for large phytoplankton silicate uptake and Si:N ratio", units="mol SiO4 kg-1", &
+                   default=2.0e-6)
+    call get_param(param_file, "generic_COBALT", "k_sio4_Md",phyto(MEDIUM)%k_sio4, &
+                   "half-saturation for medium phytoplankton silicate uptake and Si:N ratio", units="mol SiO4 kg-1", &
+                   default=1.0e-6)
+    ! Iron uptake and limitation parameters
+    call get_param(param_file, "generic_COBALT", "k_fed_Di", phyto(DIAZO)%k_fed, &
+                   "half-saturation constant for diazotroph iron uptake and growth limitation", units="mol Fed kg-1", &
+                   default=4.0e-9)
+    call get_param(param_file, "generic_COBALT", "k_fed_Lg", phyto(LARGE)%k_fed, &
+                   "half-saturation for large phytoplankton iron uptake and growth limitation", units="mol Fed kg-1", &
+                   default=2.0e-9)
+    call get_param(param_file, "generic_COBALT", "k_fed_Md", phyto(MEDIUM)%k_fed, &
+                   "half-saturation for medium phytoplankton iron uptake and growth limitation", units="mol Fed kg-1", &
+                   default=8.0e-10)
+    call get_param(param_file, "generic_COBALT", "k_fed_Sm", phyto(SMALL)%k_fed, &
+                   "half-saturation for small phytoplankton iron uptake and growth limitation", units="mol Fed kg-1", &
+                   default=4.0e-10)
+    call get_param(param_file, "generic_COBALT", "k_fe_2_n_Di", phyto(DIAZO)%k_fe_2_n, &
+                   "internal iron quota half-saturation for diazotroph growth", units="mol Fe (mol N)-1", &
+                   default=12.0e-6, scale=c2n)
+    call get_param(param_file, "generic_COBALT", "k_fe_2_n_Lg", phyto(LARGE)%k_fe_2_n, &
+                   "internal iron quota half-saturation for large phytoplankton growth", units="mol Fe (mol N)-1", &
+                   default=10.0e-6, scale=c2n)
+    call get_param(param_file, "generic_COBALT", "k_fe_2_n_Md", phyto(MEDIUM)%k_fe_2_n, &
+                   "internal iron quota half-saturation for medium phytoplankton growth", units="mol Fe (mol N)-1", &
+                   default=4.0e-6, scale=c2n)
+    call get_param(param_file, "generic_COBALT", "k_fe_2_n_Sm",phyto(SMALL)%k_fe_2_n, & 
+                   "internal iron quota half-saturation for small phytoplankton growth", units="mol Fe (mol N)-1", &
+                   default=2.0e-6, scale=c2n)
+    call get_param(param_file, "generic_COBALT", "fe_2_n_max_Di", phyto(DIAZO)%fe_2_n_max, &
+                   "maximum internal iron quota for diazotrophs",  units="mol Fe (mol N)-1", &
+                   default=500.0e-6, scale=c2n)
+    call get_param(param_file, "generic_COBALT", "fe_2_n_max_Lg", phyto(LARGE)%fe_2_n_max, &
+                   "maximum internal iron quota for large phytoplankton",  units="mol Fe (mol N)-1", &
+                   default=500.0e-6, scale=c2n)
```

Comment:
After the discussion at the COBALT Doc & Dev meeting today we thought it would be better to have the input in mol Fe (mol N)-1 which would make the default here  `default=c2n*500.0e-6`. 

We should check if this impacts the ability to exactly represent the defaults in the parameter file.